README.md

```diff
@@ -361,14 +358,12 @@ import ballerinax/docker;
 }
 
 @docker:Expose{}
-endpoint http:Listener travelAgencyEP {
-    port:9090
-};
+listener http:Listener travelAgencyEP = new(9090);
 
 // http:Client endpoint definitions to communicate with other services
 
 @http:ServiceConfig {basePath:"/travel"}
-service<http:Service> travelAgencyService bind travelAgencyEP {
+service travelAgencyService on travelAgencyEP {
 ``` 
 
 - Now you can build a Ballerina executable archive (.balx) of the service that we developed above, using the following command. This will also create the corresponding Docker image using the Docker annotations that you have configured above. Navigate to `service-composition/guide` and run the following command.
```

Comment:
Instead of `This will` shall we present the sentences in present tense`This creates`.